src/toast/Toast.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import "@clr/icons/clr-icons.min.css";
+import "@webcomponents/custom-elements/custom-elements.min.js";
+import "@clr/icons/clr-icons.min.js";
+import {Icon} from "../icon";
+import "./styles.css";
+
+export default class Toast extends React.Component<Props> {
+    constructor(props: Props) {
+        super(props);
+        props.notify(this.launch_toaster);
+        this.state = {
+            showToast: false,
+            msg: "",
+            type: "success",
```

Comment:
Please can you add enum constants for "position" and "type"?

Reply:
Addressed comment